.github/workflows/deployment.yml

```diff
@@ -89,13 +89,16 @@ jobs:
         toTag: ${{env.NEW_VERSION}}
         token: ${{ secrets.GITHUB_TOKEN }}
     - name: Create release
-      if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+      if: github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.previous_tag.outputs.tag < env.NEW_VERSION
       uses: ncipollo/release-action@v1
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        tag: ${{env.NEW_VERSION}}
+        tag: ${{ env.NEW_VERSION }}
         draft: false
-        body: ${{steps.build_changelog.outputs.changelog}}
+        body: |
+          [PyEPP on PyPI](https://pypi.org/project/pyepp/)
```

Comment:
Can you contact the version number at the end of this URL, like `https://pypi.org/project/pyepp/0.0.15/` so if someone goes back to releases, they can find the right URL for the version?

Reply:
ok, I will add it. thanks